Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/extrude.h

```diff
@@ -64,28 +64,29 @@ struct Identity_functor
 
 /**
  * \ingroup PMP_meshing_grp
- * Extrudes the open mesh input and put the result in output. The mesh generated is a closed mesh with 
- * a bottom and top part, both having the same graph combinatoric as input(except that the orientation 
- * of the faces of the bottom part are reversed). The bottom and the top parts are connected by a 
- * triangle strip between each boundary cycles. The coordinates of the points associated to the 
+ * extrudes the open surface mesh `input` and puts the result in `output`. The mesh generated is a closed 
+ * surface mesh with a bottom and top part, both having the same graph combinatorics as `input` (except 
+ * that the orientation of the faces of the bottom part is reversed). The bottom and the top parts are 
+ * connected by a triangle strip between each boundary cycles. The coordinates of the points associated to the 
```

Comment:
"each boundary cycles" should be either "boundary cycles", or "each boundary cycle" (Imho the 1st one is better)